api/src/wfl/util.clj

```diff
@@ -377,3 +379,34 @@
   [parts]
   (letfn [(make-part [[k v]] {:name (name k) :content v})]
     (map make-part parts)))
+
+(defn str->date
+  "Parse date string `s` to date object, with optional `format`."
+  ([s format]
+   (.parse (SimpleDateFormat. format) s))
+  ([s]
+   (str->date s "yyyy-MM-dd")))
+
+(defn datetime->str
+  "Parse `datetime` object into string, with optional `format`.
+
+   Note this function uses UTC as the time zone offset by default."
+  ([^Instant datetime format]
+   (let [formatter (-> format
+                       DateTimeFormatter/ofPattern
+                       (.withZone (ZoneId/from ZoneOffset/UTC)))]
+     (.format formatter datetime)))
+  ([^Instant datetime]
+   (datetime->str datetime "yyyy-MM-dd")))
+
+(defn now
+  "Return timestamp of now in ^Instant format."
+  []
+  (Instant/now))
```

Comment:
We use `OffsetDateTime/now` for our database stuff, presumably because https://stackoverflow.com/questions/58434148/what-is-better-to-persist-in-db-offsetdatetime-or-instant#:~:text=Instant%20is%20the%20simplest%2C%20simply,model%20data%20in%20simpler%20applications.
Maybe we should do the same here?

Given your usage in tests, getting the date from this instant may not be correct, especially if you're doing late night coding in the USA. See my comment below but I think we can remove much of this formatting stuff.

Reply:
I see, I forgot we used `OffsetDateTime/now` for out DB stuff, and made an assumption we want everything in UTC. Will address this.